bin/reth/src/db/mod.rs

```diff
@@ -86,6 +88,26 @@ pub struct ListArgs {
     json: bool,
 }
 
+/// Parse the table key argument to json
+pub fn maybe_json_value_parser(value: &str) -> Result<String, eyre::Error> {
+    if serde_json::from_str::<serde::de::IgnoredAny>(value).is_ok() {
+        Ok(value.to_string())
+    } else {
+        // Upgrade the raw command line primitive argument to be a valid json object
+        Ok(format!(r#""{}""#, value))
+    }
+}
+
+#[derive(Parser, Debug)]
+/// The arguments for the `reth db get` command
+pub struct GetArgs {
+    /// The table name
+    table: String, // TODO: Convert to enum
+    /// The key to get content for
+    #[arg(value_parser = maybe_json_value_parser)]
+    key: String,
+}
```

Comment:
this should be a separate module

Reply:
@mattsse  I have moved a command to separate module and added missing tests to cli parser: a62d5fb10df9ee59a3cb95dd4933439d9a0e6eab